{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Product defects","children":[{"name":"Timed out retrying after 10000ms: Expected to find element: `h1`, but never found it.","children":[{"name":"Check App1 and App2 elements","uid":"9b318e86bf5828d4","parentUid":"23fd062f6b4cc0af7a7ed54a88a82e22","status":"failed","time":{"start":1698569437466,"stop":1698569447892,"duration":10426},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"23fd062f6b4cc0af7a7ed54a88a82e22"},{"name":"The following error originated from your application code, not from Cypress.\n\n  > Cannot find module './src/components/Button.js'\nwhile loading \"./Button\" from webpack/container/reference/app2\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","children":[{"name":"Check App 1 built and running","uid":"bc5b0953292ca4c1","parentUid":"648b2475f8edd6983e414deda19cdcb4","status":"failed","time":{"start":1698569645802,"stop":1698569645804,"duration":2},"flaky":true,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"Check buttons in App 1 exist","uid":"77cda41ee1a26321","parentUid":"648b2475f8edd6983e414deda19cdcb4","status":"failed","time":{"start":1698569645804,"stop":1698569645804,"duration":0},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"648b2475f8edd6983e414deda19cdcb4"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"}],"name":"categories"}